src/java/org/apache/cassandra/tcm/serialization/Version.java

```diff
@@ -68,6 +68,8 @@ public enum Version
      */
     V7(7),
 
+    V8(8),
```

Comment:
add a comment which says what was introduced by this version, same as for other versions above